public/legacy/modules/Emails/metadata/detailviewdefs.php

```diff
@@ -50,14 +50,15 @@
                     'customCode' => '<input type=button onclick="window.location.href=\'index.php?module=Emails&action=DeleteFromImap&folder=INBOX.TestInbox&folder=inbound&inbound_email_record={$bean->inbound_email_record}&uid={$bean->uid}&msgno={$bean->msgNo}&record={$bean->id}&return_module=Emails&return_action=index\';" value="{$MOD.LBL_BUTTON_DELETE_IMAP}">'
                 ],
                 'FIND_DUPLICATES',
+                //Fix #59 - "Bad data passed in;" When trying to forward or reply to emails
```

Comment:
Hi @abuzarfaris,

Could we remove the comment like we did on the other ones please?

Reply:
@clemente-raposo Updated removed comment